database/redis/database.go

```diff
@@ -37,6 +37,14 @@ const (
 	Cli      DBSource = "Cli"
 )
 
+// Test data for configuration
+const (
+	db                  = 1
+	testSource DBSource = "test"
+)
+
+var testConfig = Config{Port: "6379", Host: "0.0.0.0", DB: db}
+
```

Comment:
My previous comment was that we should not have this config in this file. This config should appear only in test files

Reply:
fixed